src/components/Table/utils/RowStorage.ts

```diff
@@ -1,4 +1,5 @@
 import { reaction } from "mobx";
+import { loadArrayOrUndefined } from "src";
```

Comment:
**nit:** I have see issues sneak up with importing directly from `src` once we build the package. Seems to cause some circular import issues. Have you tried building and syncing this locally in BP? Maybe worth a try if not.